types/mithril/tslint.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
can you just extend "../tslint.json" and then express additional rules as diffs from that base?